### ISSUE TITLE
Part of MacOS menu doesn't disappear of the screen

### ISSUE THREAD

**Issue by rschio** [tailscale/tailscale#14245]
### What is the issue?

After selecting a Mullvad exit node using the MacOS menu bar part of the dialog doesn't disappear from screen.
<img width="926" alt="Screenshot 2024-11-28 at 15 59 28" src="https://github.com/user-attachments/assets/64613e93-6a2a-4fde-9a22-b231a098886c">

### Steps to reproduce

Activate the exit node using the MacOS extension, deactivate and activate again. It doesn't occur always, but I could reproduce many times.
Quitting and reopening the Tailscale app resolves the issue.

### Are there any recent changes that introduced the issue?

No.

### OS

macOS

### OS version

15.2 Beta (24C5089c)

### Tailscale version

1.76.6

**Comment by agottardo** [comment]
Can you try reproducing on the latest unstable version (download from https://pkgs.tailscale.com)? We just shipped a UI change there that should have resolved this. Thanks.

**Comment by rschio** [comment]
Yes, the same problem, but now the dialog is in the bottom left corner.
In the previous version sometimes the problem was occurring in the bottom left corner too.
![Screenshot 2024-11-28 at 18 46 53](https://github.com/user-attachments/assets/df1f3fa2-5b61-4dbd-9331-9d4623cd532a)

**Comment by rschio** [comment]
> Are you running any software to customize your Mac UI? 

No.

Tailscale version: 1.77.139

https://github.com/user-attachments/assets/5fe69192-e22a-474c-bb22-80606c5c3539

**Comment by agottardo** [comment]
@barnstar has a fix for this that we're hoping to land soon. Thanks for the report!

**Comment by rschio** [comment]
Thank you!
